tools/autograd/derivatives.yaml

```
@@ -1388,6 +1391,9 @@
 
 - name: linalg_eig(Tensor self) -> (Tensor eigenvalues, Tensor eigenvectors)
   self: linalg_eig_backward(grads, self, eigenvalues, eigenvectors)
+  # FIXME: Use eigenvalues, eigenvectors: linalg_eig_jvp instead (for now this is not possible due to a codegen limitation)
+  eigenvalues: linalg_eig_jvp_eigenvalues(self_t, eigenvalues, eigenvectors)
+  eigenvectors: linalg_eig_jvp_eigenvectors(self_t, eigenvalues, eigenvectors)
```

nit. you could simply do this and remove the auxiliary functions
```suggestion
  eigenvalues: std::get<0>(linalg_eig_jvp(self_t, eigenvalues, eigenvectors))
  eigenvectors: std::get<1>(linalg_eig_jvp(self_t, eigenvalues, eigenvectors))
```

---

That's better for sure :-) Fixed it.